projects/power_distribution/src/pd_fan_ctrl.c

```diff
@@ -56,7 +57,18 @@ static void prv_front_temp_to_fan_percent(uint16_t v_measured, uint8_t *fan_spee
     *fan_speed = 100;
     return;
   }
-  double ratio = s_fan_storage.ref_reading / 100.0;
+  if (v_measured == 0) {
```

Comment:
huh, do we ever divide `v_measured` by 0?

Reply:
Oh it's if you divide 0 by 1650. It shouldn't matter but I put it in there to be explicit